js/turtleactions/MeterActions.js

```diff
@@ -16,6 +16,12 @@
  * MA 02110-1335 USA.
 */
 
+/*exported setupMeterActions*/
+
+/* global rationalToFraction, _, blk, Singer, _blk, logo, TONEBPM, blocks, 
```

Comment:
Are you sure `blk` and specially `_blk` are globals? Where do they come from? Not everything which is not defined is necessarily a global. Thanks

Reply:
Yes it is global, use this [link ](https://github.com/sugarlabs/musicblocks/blob/d081fe3ef8e4ef14c503a271dd7b417f8802ac7c/js/activity.js#L3413)for your perusal. 

and _blk is I presume for translation purposes. Using _ is used throughout codebase for the same purposes.

and yes, I concur with your statement 
>Not everything which is not defined is necessarily a global.

hope I have answered your queries.

Reply:
By _ I think you mean the _ function in the `js/utils/utils.js` file. It is a function which takes in strings as arguments, I can't see how you are relating _blk to it. Also for the link you shared we can clearly see that the `errorMsg` function takes in `blk` as an argument and so does almost every other function in the `MeterActions.js` file. `blk` actually is the index of the block in `blocks.blockList` so there is no way it can be global. cc - @meganindya

Reply:
`blk` must have been passed to the methods from corresponding classes in `/blocks`. If it isn't that's an error. Certainly not a global.